app/helpers/katello/concerns/smart_proxy_helper_extensions.rb

```diff
@@ -0,0 +1,28 @@
+module Katello
+  module Concerns
+    module SmartProxyHelperExtensions
+      def disks(storage)
+        mount_points = {}
+        mount_points.default = []
+        disks = []
+        storage.each do |name, values|
+          mount = values['mounted']
+          next if mount_points.keys.include?(mount)
+          mount_points[mount] << name
+          values['header'] = "#{mount_points[mount].to_sentence} (#{values['path']} on #{values['filesystem']})"
+          values['total'] = values.delete('1g-blocks')
+          disks << values
```

Comment:
just wanted to :+1: this logic.  Initially i thought you were adding all the diskspace together, but consolidating among file systems is very clever and very useful to the user.